nectar/downloaders/threaded.py

```diff
@@ -160,6 +160,7 @@ def requests_kwargs_from_nectar_config(config):
                                          host=parsed_url.host, port=config.proxy_port)
             requests_kwargs['proxies'] = {'http': parsed_url.url, 'https': parsed_url.url}
 
+        requests_kwargs['stream'] = True # required for reading the download in chunks
```

Comment:
http://docs.python-requests.org/en/master/user/advanced/

```
If you set stream to True when making a request, Requests cannot 
release the connection back to the pool unless you consume all the 
data or call Response.close. 
```

The requests documentation, mentions that when stream=True, the connection is not released until either all the content is read, or `Response.close()` is explicitly closed.  In cases of HEAD requests the content will not be read.  If stream=True is blindly set, we'll need to ensure that `Response.close()` is always called.